### PR TITLE
fix(app-shell): feature flags for menu to not use launchdarkly

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

We have two adapters in flopflip: one for LaunchDarkly and one for our GraphQL endpoint. The first acts lazy and the latter eager. 

When we pass the menu flags to LaunchDarkly's adapter, they will be used to build a subscriptions to LaunchDarkly and then be persisted as a in the cache of the adapter. The value then cached is `false` (as the default). This means the menu item will not show unless the page is reloaded until the cache updates. This is cause the default value from the cache is used as LaunchDarkly has precedence over the MC API flags.

In short: we should not use LaunchDarkly to toggle menu visibility. In fact no team right now does. Instead we should use the MC API. For this to work eagerly (menu items show without reloading) we need to have the MC API adapter have precedence not the LaunchDarkly adapter.

### CODE REVIEW

**Review by yassinejebli — APPROVED** [review]

**Review by kark — APPROVED** [review]
Thank you 😊

**Review by CarlosCortizasCT — APPROVED** [review]
